master/internal/provisioner/gcp.go

```diff
@@ -87,6 +87,7 @@ func newGCPCluster(
 		AgentID: `$(curl "http://metadata.google.internal/computeMetadata/v1/instance/` +
 			`name" -H "Metadata-Flavor: Google")`,
 		ResourcePool: resourcePool,
+		LogOptions:   config.GCP.buildDockerLogString(),
```

Comment:
I didn't see the `agentSetupScriptConfig` and the agent setup script is updated. Did you forget to include those changes?

Reply:
nvm, I just found `LogOptions` already exists